app/decorators/user_decorator.rb

```diff
@@ -40,7 +40,7 @@ def masked_two_factor_phone_number
   end
 
   def identity_not_verified?
-    user.active_profile ? false : true
+    !user.verified?
```

Comment:
The reason for having a separate method is because the verification query goes through the UserDecorator class, or at least it should. So, when the query is moved back to the UserDecorator class (instead of the User class where it was recently moved), it would look like this:

```ruby
def identity_needs_verification?
  loa2_or_above? && !UserDecorator.new(current_user).verified?
end
```
which is harder to read IMO.